kura/distrib/src/main/resources/generic-aarch64/kura_install.sh

```diff
@@ -85,13 +85,26 @@ sed -i "s|/bin/sh KURA_DIR|/bin/bash ${INSTALL_DIR}/kura|" /lib/systemd/system/f
 systemctl daemon-reload
 systemctl enable firewall
 
-# disables cloud-init if exists and allows interface management to network-manager
+# disables cloud-init network management if exists, sets netplan network renderer to NetworkManager allowing interface management to NetworkManager
 if [ -d /etc/cloud/cloud.cfg.d ]; then
     echo "network: {config: disabled}" | sudo tee -a /etc/cloud/cloud.cfg.d/99-disable-network-config.cfg > /dev/null
 fi
+if [ -d /etc/netplan/ ]; then
+    cp /etc/netplan/00-installer-config.yaml /etc/netplan/00-installer-config.yaml.BAK
+    cat << EOF >> /etc/netplan/00-installer-config.yaml
+# This file describes the network interfaces available on your system
+# For more information, see netplan(5).
+network:
+  version: 2
+  renderer: NetworkManager
+EOF
+    netplan generate
+    netplan apply
+    systemctl restart NetworkManager
```

Comment:
I would remove this line, since the user is prompt to restart the gateways anyways. Also the firewall requires a restart, network-manager is not the only one requiring a restart.